packages/theme-default/src/components/SocialLinks/LinkContent.tsx

```diff
@@ -1,10 +1,16 @@
 import { useState } from 'react';
-import { SocialLink } from '@rspress/shared';
+import { SocialLink, SocialLinkIcon } from '@rspress/shared';
 import styles from './index.module.scss';
 import presetIcons from './presetIcons';
 
+type ExtendedSocialLink = SocialLink & { mode: 'dom', content: string | HTMLElement, icon: SocialLinkIcon };
```

Comment:
This can also be uniformly defined in shared? And we need also update the document with this type.